website/docs/getting-started/setup/installation-guides/google-cloud-run.mdx

```diff
@@ -269,22 +90,13 @@ This guide is optional, and you can skip to [Create Service](#create-service). F
 
     | Name | Value |
     |------------------------|------------------------------------------| 
-    | **FILESTORE_IP_ADDRESS** | The IP address you noted down in step 4 of [Create a Filestore instance](#create-a-filestore-instance). |
-    | **FILE_SHARE_NAME** | The file share name you noted down in step 4 of [Create a Filestore instance](#create-a-filestore-instance). |
+    | **FILESTORE_IP_ADDRESS** | The IP address you noted down in step for creating a Filestore instance in [Prerequisites]](#prereuisites). |
+    | **FILE_SHARE_NAME** | The file share name you noted down in step for creating a Filestore instance in [Prerequisites]](#prereuisites). |
     | **APPSMITH_MONGODB_URI** | Specify the connection string of the external MongoDB instance required for Appsmith data storage. <br/><br/> <i>Format</i>: <br/>`mongodb://<USERNAME>:<PASSWORD>@<HOST>:<PORT>/?authSource=<DATABASE_NAME>&replicaSet=<REPLICA_SET_NAME>[&options]`|
     | **APPSMITH_ENCRYPTION_SALT** | Specify an encryption salt to encrypt values in the database.|
     | **APPSMITH_ENCRYPTION_PASSWORD** | Specify an encryption password to encrypt values in the database.|
     | **APPSMITH_ENABLE_EMBEDDED_DB** | `0`|
-    | **APPSMITH_KEYCLOAK_DB_URL** | `jdbc:postgresql:///<DATABASE_NAME>?cloudSqlInstance=<INSTANCE_CONNECTION_NAME>&socketFactory=com.google.cloud.sql.postgres.SocketFactory&user=<POSTGRESQL_USER_NAME>&password=<POSTGRESQL_USER_PASSWORD>`|
-    | **APPSMITH_KEYCLOAK_DB_USERNAME** |`appsmith`|
-    | **APPSMITH_KEYCLOAK_DB_PASSWORD** |`appsmith`|
-    | **APPSMITH_KEYCLOAK_DB_DRIVER** |`postgresql`|
-    | **KC_DB_URL** | `jdbc:postgresql:///<DATABASE_NAME>?cloudSqlInstance=<INSTANCE_CONNECTION_NAME>&socketFactory=com.google.cloud.sql.postgres.SocketFactory&user=<POSTGRESQL_USER_NAME>&password=<POSTGRESQL_USER_PASSWORD>`|
-    | **KC_DB_USERNAME** |`appsmith`|
-    | **KC_DB_PASSWORD** |`appsmith`|
-    | **KC_TRANSACTION_XA_ENABLED** |`false`|
-    | **APPSMITH_REDIS_URL** | If you followed the steps in the [Create Redis Instance](#create-a-redis-instance) section, add the host and port from the Primary Endpoint of the Redis instance you noted down in step 5. <br/><br/> <i>Format</i>: <br/>`redis://<HOST>:<PORT>`|
-
+    
 10. Scroll down to **Cloud SQL connections** section.
```

Comment:
Cloud Sql connection is required only if sso is required
So, the points 
10
11
12
14 
points should be redirected to the page where SSO related steps are there.